doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -268,7 +268,7 @@ integration with 3rd party tools like logstash.
       #  server: 127.0.0.1
       #  port: 6379
       #  async: true ## if redis replies are read asynchronously
-      #  mode: list ## possible values: list (default), channel
+      #  mode: list ## possible values: list (default), rlist (uses RPUSH instead of LPUSH), channel
```

Comment:
I'm not sure why 'list' was named that way originally, but to me it would make sense to actually use 'lpush' and 'rpush' as names instead. Perhaps we can add that right away with backwards compatibility. Thoughts?